app/front/scripts/services/navigation.js

```diff
@@ -28,7 +28,7 @@
           filterList.push(key+'|'+value);
         });
 
-        var embed = (state.isEmbeded)? '/embed': '';
+        var embed = (state.isEmbedded)? '/embed': '';
```

Comment:
@borysyuk wouldn't it be easier to pass a var from the server to definitively say that the state is embedded? That way, it does not rely on the url path, which is fragile/could change. etc.